Stripe/legos/stripe_capture_charge/stripe_capture_charge.py

```diff
@@ -13,19 +13,16 @@ class InputSchema(BaseModel):
     )
 
 
-pp = pprint.PrettyPrinter(indent=2)
+def stripe_capture_charge_printer(output):
+    if isinstance(output, (list, tuple)):
+        pprint.pprint(output)
+    elif isinstance(output, dict):
+        pprint.pprint(output)
+    else:
+        pprint.pprint(output)
 
 
-def legoPrinter(func):
-    def Printer(*args, **kwargs):
-        output = func(*args, **kwargs)
-        print('\n')
-        pp.pprint(output)
-        return output
-    return Printer
 
-
-@legoPrinter
 def stripe_capture_charge(handle, charge_id:str) -> Any:
```

Comment:
Update the corresponding json file to have action_output_type populated too for all the legos.